qCC/db_tree/ccPropertiesTreeDelegate.cpp

```diff
@@ -400,26 +413,35 @@ void ccPropertiesTreeDelegate::fillWithMetaData(const ccObject* _obj)
 void ccPropertiesTreeDelegate::fillWithHObject(ccHObject* _obj)
 {
 	assert(_obj && m_model);
-
+	if (!_obj && !m_model)
```

Comment:
|| ?

Reply:
Darn copy and paste! :) I'll get these switched over